vispy/app/backends/_qt.py

```diff
@@ -525,13 +619,26 @@ def keyReleaseEvent(self, ev):
     def event(self, ev):
         out = super(QtBaseCanvasBackend, self).event(ev)
         t = ev.type()
+        if PYQT6_API:
```

Comment:
Similar to the keys map, maybe this could be simplified with a `qt_event_types = QtCore.QEvent.Type if PYQT6_API else QtCore.QEvent`.

Reply:
Done.